### PR TITLE
docs(demo): correct credit card mask expression

### PR DESCRIPTION
**Pull request by sean-perkins**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

## What is the current behavior?

Closes # 

The mask for the credit card is not correct: https://tinkoff.github.io/maskito/recipes/card 

## What is the new behavior?

Updates the mask expression to the expected mask input.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/pages/recipes/card/examples/1-basic/component.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class CardDocExample1 {
             ' ',
             ...Array(4).fill(/\d/),
             ' ',
-            ...Array(3).fill(/\d/),
+            ...Array(4).fill(/\d/),
```
Comment (nsbarsukov) [inline projects/demo/src/pages/recipes/card/examples/1-basic/component.ts:23]: Hello!
Thank you for your contribution!

Could you provide us with the source where we can read that card number can be 20-characters length?

I've tried to ask Google about it. It says:
> Payment card numbers are composed of 8 to 19 digits

Source: [Wikipedia](https://en.wikipedia.org/wiki/Payment_card_number#:~:text=4%20References-,Structure,bank%20identification%20number%20(BIN))

File: projects/demo/src/pages/recipes/card/examples/1-basic/component.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export class CardDocExample1 {
             ' ',
             ...Array(4).fill(/\d/),
             ' ',
-            ...Array(3).fill(/\d/),
```
Comment (vladimirpotekhin) [inline projects/demo/src/pages/recipes/card/examples/1-basic/component.ts:23]: Hi! 👋  I did not find information that a payment card can contain more than 19 digits. https://en.wikipedia.org/wiki/Payment_card_number
Am I wrong?

**Comment by nsbarsukov** [comment]
This PR should also modify this Cypress test

https://github.com/Tinkoff/maskito/blob/c3b7ee4f5a97ffe98ef6c2fa4bbd02d1b04ceb99/projects/demo-integrations/cypress/tests/recipes/card/card.cy.ts#L28-L32

File: projects/demo/src/pages/recipes/card/examples/1-basic/component.ts (hunk 1 above)
Comment (sean-perkins) [reply projects/demo/src/pages/recipes/card/examples/1-basic/component.ts:23]: The existing documentation on `main` is correct. I somehow misinterpreted the placeholder text vs. the supported mask input and thought it should accept 4 digits at the end. 

I'll close out this PR, it isn't needed 🙂

**Pull request closed without merging**